plugins/httpapi/aci.py

```diff
@@ -325,3 +328,15 @@ def cert_auth(self, method, path, payload=""):
             + "APIC-Request-Signature={0}".format(to_native(base64.b64encode(sig_signature)))
         )
         return headers
+
```

Comment:
do we need to define this function twice, or can we define in utils or something?

Reply:
I am a bit confused about this function and the need for it.

Can't set the full path in module_utils/aci.py which is always called, then add that abspath() to parameters and use that in httpapi/aci.py?